winit-core/src/window.rs

```diff
@@ -35,6 +37,8 @@ pub trait Window: HasWindowHandle + HasRawWindowHandle05 {
     fn current_monitor(&self) -> Option<MonitorId>;
 
     fn primary_monitor(&self) -> Option<MonitorId>;
+
+    fn as_any(&mut self) -> &mut dyn Any;
```

Comment:
This is not possible?

```suggestion
    fn as_any(&mut self) -> &mut dyn Any {
        self
    }
```

What if we add `Window: Any` to the trait itself?

Reply:
yeah, it's not, because it'll require sized and it's not object safe, etc. it's not really a big issue though.

Reply:
You can try to pull my code and play with it, but it wasn't really working.